src/native/physics/StepListener.cpp

```diff
@@ -1,22 +1,17 @@
-// ------------------------------------ //
```

Comment:
Please keep my formatting style intact.

Reply:
done pal, sorry about that i have a weird habit at changing that